client/scripts/controllers/server/comments.ts

```diff
@@ -6,8 +6,9 @@ import app from 'state';
 import { uniqueIdToProposal } from 'identifiers';
 
 import { CommentsStore } from 'stores';
-import { OffchainComment, OffchainAttachment, IUniqueId, AddressInfo, CommunityInfo, NodeInfo } from 'models';
+import { OffchainComment, OffchainAttachment, IUniqueId, AddressInfo, CommunityInfo, NodeInfo, OffchainThread } from 'models';
 import { notifyError } from 'controllers/app/notifications';
+import { modelFromServer as modelThreadFromServer } from 'controllers/server/threads';
```

Comment:
is there a reason importing threads controller is necessary?

Reply:
No old code good catch